lib/sles4sap_publiccloud_basetest.pm

```diff
@@ -14,40 +14,34 @@ use warnings FATAL => 'all';
 use Exporter 'import';
 use testapi;
 use qesapdeployment;
+use sles4sap_publiccloud;
 use publiccloud::utils;
 
 our @EXPORT = qw(cleanup);
 
 
 sub cleanup {
     my ($self, $args) = @_;
+
+    record_info('Cleanup',
+        join(' ',
+            'cleanup_called:', (defined($self->{cleanup_called}) ? "$self->{cleanup_called}" : 'undefined'),
+            'network_peering_present:', (defined($self->{network_peering_present}) ? "$self->{network_peering_present}" : 'undefined')));
```

Comment:
`$self->{cleanup_called}  //  'undefined'`
I think that it should work this way too.